db/model/event.js

```diff
@@ -33,6 +34,11 @@ module.exports = function event(seq, dataTypes) {
       allowNull: false,
       comment: 'This is a readable event logline',
     },
+    type: {
+      type: dataTypes.STRING(constants.fieldlen.normalName),
```

Comment:
free text here is recipe for chaos! what are the values you actually expect to see here? would it be possible to make it an enum with "Other" as one of the options? then add options as needed down the line? it's not going to be infinite! i'm sure you can come up with a half dozen values or fewer

Reply:
In this case, I think free text is actually the best option for us. The "type" is defined by the bot developer. e.g. We will have a type values of: ["sendEmailTCE", "uploadFileToCase", "pageTeam", "createSlackChannel", ...] .. If a new bot developer wants to create events of their own unique type, they should be able to do that without needing to make changes to the platform

Reply:
I agree with you, but I also still think it's a recipe for chaos... oh well